Volontulo/app/src/main/res/layout/activity_nodrawer.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <android.support.design.widget.AppBarLayout
```

Comment:
Meybe it's better to use <include app




























































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































Maybe it would be better to use <include layout="app_bar"> instead of that repeated code